ledger-service/http-json/src/main/scala/com/digitalasset/http/util/LedgerOffsetUtil.scala

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2020 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.http.util
+
+import com.digitalasset.ledger.api.v1.ledger_offset.LedgerOffset
+import scalaz.{-\/, \/, \/-}
+
+object LedgerOffsetUtil {
+
+  private val LongEitherLongLongOrdering: Ordering[Long \/ (Long, Long)] =
+    Ordering.by[Long \/ (Long, Long), (Long, Long)] {
```

Comment:
scalaz's Order will give properly disjoint results here:

```suggestion
    import scalaz.std.anyVal._
    Order[Long \/ (Long, Long)].toScalaOrdering
```